pandas/core/dtypes/common.py

```diff
@@ -823,9 +823,9 @@ def is_any_int_dtype(arr_or_dtype):
 
     This function is internal and should not be exposed in the public API.
 
-    null-able Integer support
+    .. versionchanged:: 0.24.0
 
-    .. versionadded:: 0.24.0
+       Nullable Integer support
```

Comment:
Can you put this in a full sentence? 

Something like "The nullable Integer dtypes (e.g. pandas.Int64Dtype) are also considered as integer by this function."